wagtail/admin/wagtail_hooks.py

```diff
@@ -232,65 +232,69 @@ def page_listing_buttons(page, page_perms, next_url=None):
             },
             priority=10,
         )
+
+    yield ButtonWithDropdownFromHook(
+        "",
+        hook_name="register_page_listing_more_buttons",
+        page=page,
+        page_perms=page_perms,
+        next_url=next_url,
+        icon_name="dots-horizontal",
+        attrs={
+            "target": "_blank",
+            "rel": "noreferrer",
+            "aria-label": _("More options for '%(title)s'")
+            % {"title": page.get_admin_display_title()},
+        },
+        priority=50,
+    )
+
+
+@hooks.register("register_page_listing_more_buttons")
+def page_listing_more_buttons(page, page_perms, next_url=None):
```

Comment:
Done in d7a148c9d380627b8ef866135d944a80dd3d62b0 - although this may only be a short-lived fix, as the designs call for "Add child page" to be pulled out into its own action in the header.